packages/editor/src/components/post-visibility/index.js

```diff
@@ -145,6 +147,16 @@ export class PostVisibility extends Component {
 					/>
 				</div>
 			),
+			<ConfirmDialog
+				key={ 'private-publish-confirmation' }
```

Comment:
What's this `key` used for?

Also, plain strings can also be passed as HTML attributes:

```suggestion
				key="private-publish-confirmation"
```

Reply:
> What's this key used for?

Without it, we run into a `Each child in a list should have a unique "key" prop` warning. I _think_ this happens because the visibility options are dynamically rendered via `visibilityOptions.map` but I could be wrong. 

> Also, plain strings can also be passed as HTML attributes

Good point, thank you!